package.json

```diff
@@ -38,16 +37,18 @@
     ]
```

Comment:
- [ ] It would be nice if you fix the GitHub Account handle in your readme file because it is broken right now 😨 Kindly fix it to make your readme more descriptive and professional 👍